packages/core/src/electron-browser/menu/electron-main-menu-factory.ts

```diff
@@ -74,7 +74,11 @@ export class ElectronMainMenuFactory {
         preferencesService.onPreferenceChanged(debounce(() => {
             if (this._menu) {
                 for (const item of this._toggledCommands) {
-                    this._menu.getMenuItemById(item).checked = this.commandRegistry.isToggled(item);
+                    const menu = this._menu.getMenuItemById(item);
+                    if (!menu) {
+                        continue;
+                    }
+                    menu.checked = this.commandRegistry.isToggled(item);
```

Comment:
Nitpick: You could invert the `!menu` check and run `menu.checked = this.commandRegistry.isToggled(item);` inside the branch, instead of introducing mental overhead with `continue`. It looks like you already did that in line 270.